src/commands/git.rs

```diff
@@ -1,4 +1,5 @@
 use crate::errors::*;
+use crate::errors;
```

Comment:
We're importing everything inside of the `errors` module above this line; this module-level import isn't required. :slightly_smiling_face:

Reply:
Please check below for the name shadowing workaround